generator/views/metric_definitions_view.py

```diff
@@ -169,11 +169,11 @@ def to_lookml(self, bq_client, v1_name: Optional[str]) -> Dict[str, Any]:
                     BETWEEN
                     COALESCE(
                         SAFE_CAST(
-                            {{% date_start {data_source.submission_date_column or "submission_date"} %}} AS DATE
```

Comment:
A `submission` dimension is always created pointing to `submission_date_column`, so we don't need to do the mapping here